src/colvarscript.h

```diff
@@ -254,15 +279,35 @@ extern "C" {
            if (cvm::main()->read_config_string(conf) == COLVARS_OK) {
              return COLVARS_OK;
            }
-           script->set_str_result("Error parsing configuration string");
+           script->set_error_msg("Error parsing configuration string");
            return COLVARSCRIPT_ERROR;
            )
 
+  CVSCRIPT(cv_configfile,
+           "Read configuration from a file",
+           1, 1,
+           {"conf_file (str) - Path to configuration file"},
```

Comment:
Note that I haven't implemented a handler yet for the fourth argument of `CVSCRIPT`, which would be ignored.